### DISCUSSION TITLE
Customizing Height of 3D Models in AR: Is it Possible to Set a Specific Height for Augmented Reality Placement?"

### DISCUSSION THREAD

**Discussion by abhirami100** [google/model-viewer#4641] [Q&A]
How can I modify the  height of a 3D model when it is rendered in augmented reality. I'm looking for a way to set a custom height for the model within the AR space. Are there any options or configurations available for adjusting the vertical placement of the model in the AR view?

**Comment by danilokrulj** [comment]
If you want to elevate the model to a certain height you could make an animation and stop it just before it ends. Im not sure if raising its position on the Y axis is possible in AR mode but you can try that also, maybe get the objectplaced event and do something there?

**Reply by elalish** [reply]
Indeed, we don't have a vertical offset feature yet in AR, though it has been asked for before - PRs welcome! Meanwhile, yes, you can abuse animations to achieve this. You can even give a constant transform offset in your animation, which should have this effect from the first frame. Just be aware that our framing is based on the resting pose, so you may need to add a few unreferenced vertices to make sure we keep everything in frame.
